operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -3,24 +3,28 @@
 **NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
 is only relevant for booting compute nodes and can be ignored when working with NCN images.
 
-This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image
+with minor command modifications.
 
-1. Identify the NCN image to be modified.
+1. (`ncn-mw#`) Identify the NCN image to be modified.
 
-    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs.
-    However, the steps are the same for any Management NCN SquashFS image.
+    If this procedure is being done as part of a CSM upgrade, then the documentation which linked to this procedure will have
+    provided instructions for how to set the `ARTIFACT_VERSION` variable.
 
-    If the image to be modified is the image currently booted on a Kubernetes NCN, the value for `ARTIFACT_VERSION` can be found by looking
-    at the boot parameters for the NCNs, or from `/proc/cmdline` on a booted Kubernetes NCN. The version has the form of `X.Y.Z`.
-    See: [boot parameters](../../background#metalserver)
+    Otherwise, if the image to be modified is the image currently booted on an NCN, then find the value for `ARTIFACT_VERSION`
+    by looking at the boot parameters for the NCNs, or by reading  `/proc/cmdline` on a booted NCN. The version has the form of `X.Y.Z`.
+    See: [boot parameters](../../background#metalserver).
```

Comment:
This link doesn't resolve correctly for me. It's not a fully broken link (probably why linkchecker is not seeing a problem), but `background` is a directory so `#metalserver` doesn't resolve to an anchor here.

I think what you are going for is `[boot parameters](../../background/ncn_kernel.md#metalserver)` to redirect to the section describing the `metal.server` kernel param.

I also kind of think you should just put an example kernel command line here and show exactly where in it the X.Y.Z version is. But more important is the link not working.

Reply:
Completely agree with all of the above. Like I said in response to one of John's comments, this part of the procedure I left unchanged, as it does not apply to the CSM upgrade case. I was specifically targeting that situation in order to limit the scope of the work.

I will at least make the low hanging fruit fixes here (correcting the broken link), but I will likely defer some of this to a future ticket. I'll update with the ticket number when I have it.

Reply:
Good with me. If you want to just fix the link I'll for sure approve the PR.

As an aside I think when I was first reviewing this I did not notice that this PR was going into the `CASMINST-5480-1.3` branch, thought it was going directly into `release/1.3`.

Reply:
I've opened [CASMINST-5588](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5588) for this work. It will also cover [John's comment](https://github.com/Cray-HPE/docs-csm/pull/2790/files#r1014393899)

Reply:
```suggestion
See [`metal.server` boot parameter](../../background/ncn_kernel.md#metalserver).
```